documentation/library_guide/parallel_api/pass_data_algorithms.rst

```diff
@@ -81,47 +76,65 @@ the buffer were created for the same queue. For example:
     return 0;
   }
 
-Alternatively, use ``std::vector`` with a USM allocator. For example:
+When using device USM, such as allocated by ``malloc_device``, manually copy data to this memory
```

Comment:
It seems the sentence might lead to a false judgment - to do manually copy to device and copy back, which is not entirely true for arbitrary use case. And even incorrect  for the case above (it is not  needed to copy data to device allocated memory for `std::fill` algorithm).

Reply:
Applied the rewrite I mention above.